From 7387c3d609e5d97906d3f0cb8f2c62c6c5a8df9b Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Tue, 20 Sep 2016 10:48:50 -0700
Subject: [PATCH 29/30] Start user session by default for logins.

Our system-auth and common-auth are either not getting called through
the dependency chain or are not including pam_systemd.so to start a
user session.

We only want a user session when it's really needed, so add it right
in this file to avoid getting user sessions on other vectors.
diff
---
 src/login/systemd-user.m4 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/login/systemd-user.m4 b/src/login/systemd-user.m4
index f188a8e..9f595bb 100644
--- a/src/login/systemd-user.m4
+++ b/src/login/systemd-user.m4
@@ -10,3 +10,4 @@ session  required pam_selinux.so nottys open
 )m4_dnl
 session  required pam_loginuid.so
 session  include system-auth
+-session optional pam_systemd.so
-- 
2.10.0

